doc/api/errors.md

```diff
@@ -883,60 +879,57 @@ An invalid value has been specified for an HTTP/2 setting.
 <a id="ERR_HTTP2_INVALID_STREAM"></a>
 ### ERR_HTTP2_INVALID_STREAM
 
-Used when an operation has been performed on a stream that has already been
-destroyed.
+An operation has been performed on a stream that has already been destroyed.
 
 <a id="ERR_HTTP2_MAX_PENDING_SETTINGS_ACK"></a>
 ### ERR_HTTP2_MAX_PENDING_SETTINGS_ACK
 
 Whenever an HTTP/2 `SETTINGS` frame is sent to a connected peer, the peer is
 required to send an acknowledgement that it has received and applied the new
-SETTINGS. By default, a maximum number of un-acknowledged `SETTINGS` frame may
+`SETTINGS`. By default, a maximum number of unacknowledged `SETTINGS` frames may
 be sent at any given time. This error code is used when that limit has been
 reached.
 
 <a id="ERR_HTTP2_NO_SOCKET_MANIPULATION"></a>
 ### ERR_HTTP2_NO_SOCKET_MANIPULATION
 
-Used when attempting to directly manipulate (e.g read, write, pause, resume,
-etc.) a socket attached to an `Http2Session`.
+A socket attached to an `Http2Session` may not be directly manipulated
+(e.g read, write, pause, resume, etc.).
 
 <a id="ERR_HTTP2_OUT_OF_STREAMS"></a>
 ### ERR_HTTP2_OUT_OF_STREAMS
 
-Used when the maximum number of streams on a single HTTP/2 session have been
-created.
+The maximum number of streams on a single HTTP/2 session have been created.
 
 <a id="ERR_HTTP2_PAYLOAD_FORBIDDEN"></a>
 ### ERR_HTTP2_PAYLOAD_FORBIDDEN
 
-Used when a message payload is specified for an HTTP response code for which
-a payload is forbidden.
+A message payload was specified for an HTTP response code for which a payload is
```

Comment:
`s/is/was/`?

Reply:
In this case, I think `is` is more idiomatic/correct because it's about the HTTP response code, not the payload. Did all the other ones so far, though. Will push a mega-verb-tense-nit commit soon! Thanks!